appeals/web/src/server/lib/display-page-formatter.js

```diff
@@ -339,3 +345,15 @@ export function formatListOfAddresses(arrayOfAddresses) {
 	}
 	return '<span>None</span>';
 }
+
+/**
+ * @param {string|number|null|undefined} appealId
+ * @returns {string}
+ */
+export function generateHorizonAppealUrl(appealId) {
```

Comment:
I added this function before I realised the Horizon appeal ID isn't available, but it's useful as a reference, and in case we can fix this in future